store/tikv/client_batch.go

```diff
@@ -501,6 +519,21 @@ func sendBatchRequest(
 	}
 	ctx1, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
+	established := atomic.LoadInt32(&batchConn.estDone) == 1
```

Comment:
Seems we don't need to wait all connections ready? I guess we don't need `estDone` and `estReady`.